client/components/ContextMenu.jsx

```diff
@@ -0,0 +1,114 @@
+/** @jsx h */
+import {h} from 'preact';
+import cls from 'classnames';
+import ContextMenuItem from './ContextMenuItem';
+import PureComponent from '../lib/PureComponent';
+import {store} from '../store';
+import {elementIsOutside} from '../utils';
+
+import s from './ContextMenu.css';
+
+export default class ContextMenu extends PureComponent {
+  componentDidMount() {
+    this.boundingRect = this.node.getBoundingClientRect();
+  }
+
+  componentDidUpdate(prevProps) {
+    if (this.props.visible && !prevProps.visible) {
+      document.addEventListener('mousedown', this.handleDocumentMousedown, true);
+    } else if (prevProps.visible && !this.props.visible) {
+      document.removeEventListener('mousedown', this.handleDocumentMousedown, true);
+    }
+  }
+
+  render() {
+    const {visible} = this.props;
+    const containerClassName = cls({
+      [s.container]: true,
+      [s.hidden]: !visible
+    });
+    const multipleChunksSelected = store.selectedChunks.length > 1;
+    return (
+      <ul ref={this.saveNode} className={containerClassName} style={this.getStyle()}>
+        <ContextMenuItem disabled={!multipleChunksSelected}
```

Comment:
This is a bit of a digression, but I think it would be really great if this markup generated here was more screen reader accessible. The main challenge seems to not be in the webpack-bundle-analyzer-specific controls, but in making the whole underlying visualization accessible, such that a user could even get to a point where the controls are meaningful to use (curious if anyone does know of a common screen reader that's able to generate a meaningful result for the treemap content itself, it seems like a hard challenge).